framework/caching/FileCache.php

```diff
@@ -155,7 +155,12 @@ protected function setValue($key, $value, $duration)
                 $duration = 31536000; // 1 year
             }
 
-            return @touch($cacheFile, $duration + time());
+            if (@touch($cacheFile, $duration + time())) {
+                clearstatcache();
```

Comment:
We've ported it Yii3 cache and while doing so, decided that it's better to clear cache for that file only via `clearstatcache(false, $cacheFile)`. Does that make sense to you?

Reply:
Stat cache is stored only for one (last) file, so I don't think you need to specify file path here. It is only used if you want to clear realpath cache, which is no needed here. So I don't think these changes make any sense.